eos_data_distribution/ndn/base.py

```diff
@@ -115,6 +115,11 @@ def __init__(self, name, face=None):
         self._callbackCount = 0
         self._responseCount = 0
 
+        if auto: self.start()
```

Comment:
got your other comment, you are right, left start in consumer/producer, we can decide to remove it later.